sdk/storage/test/test_base.cpp

```diff
@@ -132,4 +132,11 @@ namespace Azure { namespace Storage { namespace Test {
     }
   }
 
+  std::vector<uint8_t> ReadBodyStream(Azure::Core::Http::BodyStream* stream)
+  {
+    std::vector<uint8_t> body(stream->Length(), '\x00');
```

Comment:
nit: Can you tell me why we need to assign a default value?

Reply:
For example, when in some test case, we download a 4MB blob and then download the same blob of range 0~3MB. it's possible that we download the data into the same memory address.

If we don't reset the memory, we don't know if the downloaded data is right.